src/session/content/chat_action_bar.rs

```diff
@@ -452,7 +452,8 @@ impl ChatActionBar {
             let chooser = gtk::EmojiChooser::new();
             chooser.set_parent(parent);
             chooser.connect_emoji_picked(clone!(@weak self as obj => move |_, emoji| {
-                obj.imp().message_entry.insert_at_cursor(emoji);
+                // TODO: soon tdlib will support emojis with that symbol at the end
+                obj.imp().message_entry.insert_at_cursor(emoji.trim_end_matches('\u{fe0f}'));
```

Comment:
Maybe I'm just out of the loop, but could it make sense to clarify in the comment what the purpose of "that symbol" is?

Reply:
That symbol inserted automatically by gtk. I don't know what it means

Reply:
It seems to just say that the preceding character should be displayed as an emoji and not text: https://stackoverflow.com/a/38100803/9315690

Reply:
This should not be needed anymore since we updated to latest tdlib.